packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -155,31 +171,57 @@ const baseConfig = (target) => {
                     path: buildDir
                 },
                 resolve: {
+                    ...(pkg?.ccExtensibility?.extends && pkg?.ccExtensibility?.overridesDir
+                        ? {
+                              plugins: [
+                                  new OverridesResolverPlugin({
+                                      extends: [pkg?.ccExtensibility?.extends],
+                                      overridesDir: pkg?.ccExtensibility?.overridesDir,
+                                      projectDir: process.cwd()
+                                  })
+                              ]
+                          }
+                        : {}),
```

Comment:
pull the customer resolver plugin in conditionally, a no-op if both of `pkg?.ccExtensibility?.extends` and `pkg?.ccExtensibility?.overridesDir` are not defined in the users `package.json`